src/js/tabs/pid_tuning.js

```diff
@@ -2412,7 +2412,7 @@ TABS.pid_tuning.updateRatesLabels = function() {
                 );
             }
             // then display them on the chart
-            for (let i = 0; i <= balloons.length; i++) {
+            for (let i = 0; i < balloons.length; i++) {
```

Comment:
These _could_ easily be done as `for ( of )`:

```
for (balloon of balloons) {
    balloon.balloon();
}
```

same for the above.

Reply:
😜 I've tried but then the last element is still undefined.

![image](https://user-images.githubusercontent.com/8344830/102420626-db66ed00-4002-11eb-8a3a-838a7a39d713.png)

Reply:
Let me change it - we both forgot to initialize the balloon 🤣

Reply:
Tested and fixed